tests/catboost/test_catboost_model_export.py

```diff
@@ -303,22 +303,27 @@ def test_log_model_with_pip_requirements(reg_model, tmpdir):
     req_file.write("a")
     with mlflow.start_run():
         mlflow.catboost.log_model(reg_model.model, "model", pip_requirements=req_file.strpath)
-        _assert_pip_requirements(mlflow.get_artifact_uri("model"), ["mlflow", "a"])
+        _assert_pip_requirements(mlflow.get_artifact_uri("model"), ["mlflow", "a"], strict=True)
```

Comment:
How `strict` works:

https://github.com/mlflow/mlflow/blob/6a6653cb0e5b34d961e889573d6567faebea39e8/tests/helper_functions.py#L345-L353

Reply:
If `pip_requirements` is supplied, the logged requirements must be **equal** to `["mlflow"] + pip_requirements`. By setting `strict` to `True`, we can check that.